routes/web.php

```diff
@@ -1,7 +1,13 @@
 <?php
 
 use Illuminate\Support\Facades\Route;
+use App\Http\Controllers\PageController;
+use App\Http\Controllers\Authcontroller;
+use App\Http\Controllers\TaskController;
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/', [PageController::class, 'index']);
+Route::get('/login', [Authcontroller::class, 'showLoginForm'])->name('login');
+Route::post('/login', [Authcontroller::class, 'login']);
```

Comment:
Instead of these
Route::get('/login', [Authcontroller::class, 'showLoginForm'])->name('login');
Route::post('/login', [Authcontroller::class, 'login']);
Route::get('/register', [Authcontroller::class, 'showRegisterForm'])->name('register');



Do this




Route::group(['middleware'=> 'guest'], function () {
    Route::view('/login', 'auth/login')->name('login');
    Route::view('/register', 'auth/register')->name('register');
    Route::post('/login', [Authcontroller::class, 'login']);
});